.gitignore

```diff
@@ -36,6 +36,7 @@ docker/**/fhir/log/*.log.gz
 
 docker/secrets/*.pem
 docker/.env
+docker/docker-compose-internal.yml
```

Comment:
consider using `docker-compose.override.yml`. The "override" file is used automatically if it exists and does not need to be defined via `-f foo.yml`

Reply:
split in 807b846